src/components/OrderDetails.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <ion-card>
+    <ion-item lines="none" >
+      <ion-label>
+        <h1>{{order.customerName}}</h1>
+        <p>{{order.orderId}}</p>
+      </ion-label>
+      <ion-note slot="end">{{ order.orderDate }}</ion-note>
+    </ion-item>
+    <ion-item lines="full" v-for="item in order.items" :key="item.orderId" :item="item" >
```

Comment:
`key` should be unique and orderId must be common for all the items. Please update it with the right field